packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [5.3.3]
+* Update the fastlane file to fix compatibility issues with the latest flutter stable
```

Comment:
Please remove the [5.3.3] version and move the note to version [5.3.2], since it's not released yet

Reply:
Done